code/datums/keybindings/client.dm

```diff
@@ -0,0 +1,42 @@
+/datum/keybinding/client
+	category = KB_CATEGORY_UNSORTED
+
+/datum/keybinding/client/admin_help
+	name = "Admin Help"
+	keys = list("F1")
+
+/datum/keybinding/client/admin_help/down(client/C)
+	. = ..()
+	C.adminhelp()
+
+/datum/keybinding/client/ooc
+	name = "OOC"
+	keys = list("F2")
+
+/datum/keybinding/client/ooc/down(client/C)
+	. = ..()
+	C.ooc()
+
+/datum/keybinding/client/say
+	name = "Say"
+	keys = list("F3")
+
+/datum/keybinding/client/say/down(client/C)
+	. = ..()
+	C.mob?.say_wrapper()
```

Comment:
I believe a player always has a mob attached to his client. Though I see that the code already checks this a couple of times prior.